src/tbb/thread_request_serializer.h

```diff
@@ -39,6 +39,7 @@ class thread_request_serializer : public thread_request_observer {
 public:
     thread_request_serializer(thread_dispatcher& td, int soft_limit);
     void set_active_num_workers(int soft_limit);
+    int num_workers_requested() { return my_total_request.load(std::memory_order_acquire); }
```

Comment:
Happens-before relationships are not built on top of `my_total_request`, thus all the operations should be `std::memory_order_relaxed`.

Reply:
Changed.